dbt_subprojects/dex/models/automated_trades/dex_mapping.sql

```diff
@@ -0,0 +1,475 @@
+{{ config(
+        schema = 'dex',
+        alias = 'mapping',
+        materialized = 'table',
+        file_format = 'delta',
+        unique_key = ['blockchain', 'factory'],
+        post_hook='{{ expose_spells(\'["ethereum", "arbitrum", "polygon"]\',
+                                "sector",
+                                "dex",
+                                \'["0xBoxer"]\') }}'
+        )
+}}
+
+
+
+
+SELECT
+    src
+    , dex_type
+    , blockchain
+    , from_hex(factory) as factory
```

Comment:
any reason you use `from_hex()` and force strings on the values list below?
should be able to remove single quotes on address and avoid this function

Reply:
makes it easier to go from csv to sql file and the other way around